web/react/components/access_history_modal.jsx

```diff
@@ -57,6 +57,10 @@ module.exports = React.createClass({
                 newDate = (<div> {currentHistoryDate.toDateString()} </div>);
             }
 
+            if (!currentAudit.session_id && currentAudit.action.replace("/api/v1", "") === "/users/login") {
```

Comment:
Is there a reason for the strange replace compare here? Also if we do ever move to v2 of the api this will break. Maybe search for a suffix?